test/test_datasets.py

```diff
@@ -47,8 +47,8 @@ def test_openfire(self):
         with Path(tempfile.TemporaryDirectory().name) as root:
 
             # Working case
-            train_set = datasets.OpenFire(root=root, train=True, download=True)
-            test_set = datasets.OpenFire(root=root, train=False, download=True)
+            train_set = datasets.OpenFire(root=root, train=True, download=True, num_samples=200)
+            test_set = datasets.OpenFire(root=root, train=False, download=True, num_samples=200)
```

Comment:
Better to add `num_samples = 200` as a test variable above line 47 and pass it to the appropriate functions.
So that if we want to increase/decrease that value, only one line has to be changed

Reply:
Agreed!